content/_partners/business_inquiries._en.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
May consider updating the file path to match the URL path (`business_inquiries._en.md` ➡️ `business-inquiries._en.md`)

Reply:
done 🙌